# This service is used to retrieve an existing image thumbnail from an existing
# target on the interoperability server.

# Target ID.
uint64 id

---

# Whether the request was successful.
bool success

# Image thumbnail.
sensor_msgs/Image image
